packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -4485,4 +4485,183 @@ QUnit.module('Accessibility', () => {
+        });
+
+        QUnit.test(`Group body element should have correct aria if collapsibleGroups: ${collapsibleGroups}`, function(assert) {
+            const $element = $('#list').dxList({
+                dataSource: [{ key: 1, items: [1] }],
+                grouped: true,
+                collapsibleGroups,
+            });
+
+            const $groupBody = $element.find(`.${LIST_GROUP_BODY_CLASS}`);
+
+            const expectedGroupBodyAria = {
+                role: collapsibleGroups ? 'listbox' : undefined,
+                'aria-labelledby': collapsibleGroups ? true : false,
+            };
+
+            assert.strictEqual($groupBody.attr('role'), expectedGroupBodyAria.role, 'role is correct');
+            assert.strictEqual(!!$groupBody.attr('aria-labelledby'), expectedGroupBodyAria['aria-labelledby'], 'aria-labelledby is correct');
+        });
+
+        QUnit.test(`Grouped list items element should have correct aria if collapsibleGroups is changed in runtime, init value: ${collapsibleGroups}`, function(assert) {
+            const $element = $('#list').dxList({
+                dataSource: [{ key: 1, items: [1] }],
+                grouped: true,
+                collapsibleGroups,
+            });
+            const instance = $element.dxList('instance');
+
+            instance.option('collapsibleGroups', !collapsibleGroups);
+
+            const $listItems = $element.find(`.${LIST_ITEMS_CLASS}`);
+
+            const expectedListItemsAria = {
+                role: collapsibleGroups ? 'listbox' : undefined,
+                'aria-label': collapsibleGroups ? 'Items' : undefined,
+            };
+
+            assert.strictEqual($listItems.attr('role'), expectedListItemsAria.role, 'role is correct');
+            assert.strictEqual($listItems.attr('aria-label'), expectedListItemsAria['aria-label'], 'aria-label is correct');
+        });
+
+        QUnit.test(`Group element should have correct aria if collapsibleGroups is changed in runtime, init value: ${collapsibleGroups}`, function(assert) {
+            const $element = $('#list').dxList({
+                dataSource: [{ key: 1, items: [1] }],
+                grouped: true,
+                collapsibleGroups,
+            });
+            const instance = $element.dxList('instance');
+
+            instance.option('collapsibleGroups', !collapsibleGroups);
+
+            const $groupElement = $element.find(`.${LIST_GROUP_CLASS}`);
+
+            const expectedGroupAria = {
+                role: collapsibleGroups ? 'group' : undefined,
+                'aria-labelledby': collapsibleGroups ? true : false,
+            };
+
+            assert.strictEqual($groupElement.attr('role'), expectedGroupAria.role, 'role is correct');
+            assert.strictEqual(!!$groupElement.attr('aria-labelledby'), expectedGroupAria['aria-labelledby'], 'aria-labelledby is correct');
```

Comment:
It'd be better to test real ids, don't see stoppers for this. 
Not it's difficult to read and can be broken (low probability)